pkg/proxy/iptables/proxier.go

```diff
@@ -282,6 +282,12 @@ func NewProxier(ctx context.Context,
 	if err != nil {
 		logger.Error(err, "Failed to create nfacct runner")
 	}
+	if !nfacctRunner.Present() {
+		nfacctRunner = nil
+		logger.Error(err, "No support for nfacct sub-system, nfacct based metrics "+
+			"('kubeproxy_iptables_localhost_nodeports_accepted_packets_total', 'kubeproxy_iptables_ct_state_invalid_dropped_packets_total')"+
+			" won't be available.")
```

Comment:
We shouldn't hard-code that list of metrics here... it's going to end up out-of-date.
If you can't auto-generate it somehow then it may be better to just not bother to list the specific metrics.